### DISCUSSION TITLE
Installation script fails for Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by Michael-Kempe** [rabbitmq/rabbitmq-website#1499] [General]
The script I'm using was taken from _'Installing on Debian and Ubuntu'_ : [Quick Start Script](https://www.rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith).
The script failed with the following error:

```
The following packages have unmet dependencies:
 erlang-public-key : Depends: erlang-crypto (= 1:25.1.1-1rmq1ppa1~ubuntu18.04.1) but it is not installable
 erlang-snmp : Depends: erlang-crypto (= 1:25.1.1-1rmq1ppa1~ubuntu18.04.1) but it is not installable
 erlang-ssl : Depends: erlang-crypto (= 1:25.1.1-1rmq1ppa1~ubuntu18.04.1) but it is not installable
 rabbitmq-server : Depends: erlang-crypto (>= 1:25.0) but it is not installable or
                            esl-erlang (>= 1:25.0) but it is not installable
E: Unable to correct problems, you have held broken packages.
```

I have run the script on a clean Ubuntu installation.

It would be great if someone could help me with the installation.

**Comment by michaelklishin** [comment]
Erlang 25 is very unlikely to have Ubuntu 18.04 packages because modern versions depend on OpenSSL 1.1+ and Ubuntu 18 only provides 1.0.x.

Use Ubuntu 22.04.

**Reply by Michael-Kempe** [reply]
I'm using Ubuntu 22.04. Or am I misunderstanding something?

**Reply by Michael-Kempe** [reply]
Sorry, my bad.  The script contains 'bionic´ - release.' When using 'jammy' everything works!
